dag_in_context/src/lib.rs

```diff
@@ -113,28 +113,56 @@ pub(crate) fn print_with_intermediate_vars(termdag: &TermDag, term: Term) -> Str
 }
 
 // Returns a formatted string of (union call body) for each pair
-#[allow(dead_code)]
 fn print_function_inlining_pairs(
     function_inlining_pairs: Vec<function_inlining::CallBody>,
     printed: &mut String,
     tree_state: &mut TreeToEgglog,
     term_cache: &mut HashMap<Term, String>,
 ) -> String {
-    // Get unions
+    // Get unions and mark each call as inlined for extraction purposes
+    let inlined_call = "(relation InlinedCall (String Expr))";
```

Comment:
can you put this somewhere in a .egg file, along with the loop estimate table?

Reply:
I think it might be better to keep it here, since then it is by its usage. It shouldn't be used for any other optimization. `LoopNumItersGuess`, for example, is in the same .egg file as its usage.

Reply:
I do think it would be better to put this whole function into function_inlining.rs, though.